src/commonMain/kotlin/app/cash/turbine/channel.kt

```diff
@@ -87,9 +87,9 @@ public suspend fun <T> ReceiveChannel<T>.awaitEvent(name: String? = null): Event
       Event.Item(item)
     }
   } catch (e: TimeoutCancellationException) {
-    throw AssertionError("No ${"value produced".qualifiedBy(name)} in $timeout")
+    throw TurbineAssertionError("No ${"value produced".qualifiedBy(name)} in $timeout", e)
   } catch (e: TurbineTimeoutCancellationException) {
-    throw AssertionError("No ${"value produced".qualifiedBy(name)} in $timeout")
+    throw TurbineAssertionError("No ${"value produced".qualifiedBy(name)} in $timeout", e)
```

Comment:
Ahh, because we `override val cause` in `TurbineAssertionError` 🤔

Reply:
Built in one is coming now that Java 8 is the minimum: https://youtrack.jetbrains.com/issue/KT-40728